### PR TITLE
Dp 5669

### PR DESCRIPTION
**Pull request by sharku11**

### CODE REVIEW

File: src/decorators/index.test.js [hunk 1]
```diff
@@ -101,15 +101,15 @@ describe('decorators', () => {
     },
     {
       text: 'http://dialog.chat/ - test this',
-      result: [{ start: 0, end: 19, replace: 'http://dialog.chat/' }],
+      result: [{ start: 0, end: 18, replace: 'http://dialog.chat' }],
```
Comment (nkt) [inline src/decorators/index.test.js]: this is a strange fix

File: src/decorators/index.test.js [hunk 2]
```diff
@@ -101,15 +101,15 @@ describe('decorators', () => {
...
     },
     {
       text: '(test: https://dlg.im)',
       result: [{ start: 7, end: 21, replace: 'https://dlg.im' }],
     },
     {
       text: '(https://dlg.im)',
-      result: [{ start: 1, end: 15, replace: 'https://dlg.im' }],
+      result: [{ start: 0, end: 14, replace: 'https://dlg.im' }],
```
Comment (nkt) [inline src/decorators/index.test.js]: this is wrong

File: src/decorators/index.test.js (hunk 1 above)
Comment (sharku11) [reply src/decorators/index.test.js]: returned it to original state

File: src/decorators/index.test.js (hunk 2 above)
Comment (sharku11) [reply src/decorators/index.test.js]: returned it to original state

File: package.json [hunk 3]
```diff
@@ -33,14 +33,15 @@
   "homepage": "https://github.com/dialogs/markdown#readme",
   "devDependencies": {
     "@babel/core": "7.6.2",
+    "@babel/plugin-transform-named-capturing-groups-regex": "^7.8.3",
```
Comment (meafmira) [inline package.json:36]: Use exact versions everywhere

Comment (sharku11) [reply package.json]: fixed

Comment (meafmira) [reply package.json]: @sharku11 in ur commit https://github.com/dialogs/markdown/pull/21/commits/2d931952300b3f02c169685a3f9fcf1205ebc11a lockfile wasn't updated. Did u update it?

Comment (sharku11) [reply package.json]: i did "npm i" after changing packaje.json and lockfile hasn't changed

**Pull request merged**